extern/storage-sealing/precommit_policy_test.go

```diff
@@ -60,17 +59,14 @@ func fakePieceCid(t *testing.T) cid.Cid {
 func TestBasicPolicyEmptySector(t *testing.T) {
 	cfg := fakeConfigGetter(nil)
 	h := abi.ChainEpoch(55)
-	pBoundary := abi.ChainEpoch(0)
 	pBuffer := abi.ChainEpoch(2)
-	pcp := sealing.NewBasicPreCommitPolicy(&fakeChain{h: h}, cfg, pBoundary, pBuffer)
+	pcp := sealing.NewBasicPreCommitPolicy(&fakeChain{h: h}, cfg, pBuffer)
 	exp, err := pcp.Expiration(context.Background())
 
 	require.NoError(t, err)
 
 	// as set when there are no deal pieces
-	expected := h + policy.GetMaxSectorExpirationExtension() - (pBuffer * 2)
-	// as set just before returning within Expiration()
-	expected += miner.WPoStProvingPeriod - (expected % miner.WPoStProvingPeriod) + pBoundary - 1
+	expected := h + policy.GetMaxSectorExpirationExtension() - pBuffer
```

Comment:
I don't know why these were subtracting out pBuffer * 2, since that isn't what the cc lifetime method seems to do.

Reply:
Yeah, my implementation is definitely wrong here, hidden by the chosen constants. Only rationale I can come with here was to match [this](https://github.com/filecoin-project/lotus/pull/6803/files#diff-e5e6c18c093f7eb4f4d0bb2e19388a831f34ded5ceeff4737091f6dc686aafe6R183) but it's external to the Policy entirely. Not sure why I landed with that. [I'm sorry.](https://github.com/filecoin-project/lotus/pull/6803/files#r675107394)